src/code/z_demo.c

```diff
@@ -1779,10 +1801,13 @@ void Cutscene_ProcessScript(PlayState* play, CutsceneContext* csCtx, u8* script)
         return;
     }
 
+#ifdef OOT_DEBUG
+    // With debug features enabled, pressing D-pad Right on controller 1 will stop the cutscene early and skip it
```

Comment:
At first I thought "with debug features enabled" refered to an additional check by the game like a previous combo or something, but I guess you mean "if debug build"
I think being inside a `#ifdef OOT_DEBUG` is enough for documenting it's a debug feature.
Or if you really want to hammer it in just using `// Debug: description...` would be enough

------

I wouldn't use "skip" here because it's more "interrupt the cutscene" right? rather than force running the destination which looks more like what you'd expect from a skip

------

Also a style thing I prefer writing debug features docs in the imperative, like:

```
// Press D-pad Right on controller 2 to stop playing the current cutscene.
```

Since it's for the programmer reading the source. It's also usually more concise.
But may be just-me

Reply:
Ill omit the comments for now, maybe in the future we can standardize and document the various debug features in the game